src/applications/static-pages/facilities/FacilityAppointmentWaitTimesWidget.jsx

```diff
@@ -58,9 +60,9 @@ export class FacilityAppointmentWaitTimesWidget extends React.Component {
           </p>
           <div className="usa-grid-full">
             <div className="vads-u-display--flex">
-              {serviceExists.new &&
+              {serviceExists.new >= 0 &&
```

Comment:
Are we sure the value is never `null`? null >= 0 is true. Just want to check edge conditions.

Reply:
Good callout.

Reply:
I'm not sure of that myself. We can add an additional check here that it's a number to be safe and then extract the full "is number and is number greater than or equal to zero" check to a separate function.

Reply:
Added a check to make sure it's `number` before checking the value being greater or equal to 0.